devtools/conda-envs/adapters.yaml

```diff
@@ -30,6 +30,7 @@ dependencies:
   - plotly >=4.0.0
   - pyarrow >=0.13.0
   - tqdm
+  - pint <=0.9
```

Comment:
Tests unrelated to this PR fail due to pint 0.10

Reply:
Whats the pint issue?

Reply:
https://travis-ci.org/MolSSI/QCFractal/jobs/635484207?utm_medium=notification&utm_source=github_status

Reply:
```
>>> qcel.constants.conversion_factor("hartree", "bohr * elementary_charge")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/daniel/Gits/QCElemental/qcelemental/physical_constants/context.py", line 272, in conversion_factor
    return self.ureg.convert(factor, base_unit, conv_unit)
  File "/Users/daniel/miniconda3/envs/qcf/lib/python3.7/site-packages/pint/registry.py", line 884, in convert
    return self._convert(value, src, dst, inplace)
  File "/Users/daniel/miniconda3/envs/qcf/lib/python3.7/site-packages/pint/registry.py", line 1665, in _convert
    return super()._convert(value, src, dst, inplace)
  File "/Users/daniel/miniconda3/envs/qcf/lib/python3.7/site-packages/pint/registry.py", line 1279, in _convert
    return super()._convert(value, src, dst, inplace)
  File "/Users/daniel/miniconda3/envs/qcf/lib/python3.7/site-packages/pint/registry.py", line 917, in _convert
    raise DimensionalityError(src, dst, src_dim, dst_dim)
pint.errors.DimensionalityError: Cannot convert from 'hartree' ([length] ** 2 * [mass] / [time] ** 2) to 'bohr * elementary_charge' ([current] * [length] * [time])
```

Did this ever work? This seems more like an elemental issue.

Reply:
In the end, the issue was due to the pint-import-avoidance strategy.